README.md

```diff
@@ -184,6 +184,8 @@ the first available will be chosen, but you can force a specific one:
 let test#python#runner = 'pytest'
 " or
 let test#python#runner = 'nose'
+" or
+let test#python#runner = 'django'
```

Comment:
Is there the also a problem in detecting a Django project? Does every Django project have a `manage.py` file? Because I intended that here are listed only the ambigous runners.